docs/api.md

```diff
@@ -496,6 +496,93 @@ await webcontainerInstance.fs.writeFile(
 );
 ```
 
+### ▸ `watch`
+
+Watch for changes to a given file or directory.
+
+<h4 id="watch-signature">
+  <a id="watch-signature">Signature</a>
+  <a href="#watch-signature" class="header-anchor" aria-hidden="true">#</a>
+</h4>
+
+`watch(path: string, options: Options, Callback): Watcher`
```

Comment:
```suggestion
`watch(path: string, options: Options, callback: Callback): Watcher`
```

Oh and I think we're missing a definition for the type `Callback`

Reply:
Ah. there is a definition, it's just called `Listener` and this needs to be renamed